audio.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+import simpleaudio as sa
+import scales
+from typing import Dict, List
+import keyword
```

Comment:
I like to sort imports as follows:
One block of built in imports
One block of 3rd party imports
One block of my own 1st party modules.

In this case:
```
from typing import Dict, List

import numpy as np
import simpleaudio as sa

import scales
```

(the keyword import seems unused? What is it?)